src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -245,6 +245,35 @@ def set_reboot_pending(self, is_reboot_pending):
         self.is_reboot_pending = is_reboot_pending
     # endregion
 
+    # region - Terminal state management
+    def report_sequence_number_changed_termination(self):
+        """ Based on the current operation, sends """
+        current_operation = self.execution_config.operation.lower()
+        error_code = Constants.PatchOperationErrorCodes.NEWER_OPERATION_SUPERSEDED
+        message = "Execution was stopped due to a newer operation taking precedence."
+
+        if self.execution_config.exec_auto_assess_only:
+            self.add_error_to_status(message, error_code, current_operation_override_for_error=Constants.ASSESSMENT)
+            self.set_assessment_substatus_json(status=Constants.STATUS_ERROR)
+            return
+
+        if current_operation == Constants.CONFIGURE_PATCHING.lower() or \
+                current_operation == Constants.CONFIGURE_PATCHING_AUTO_ASSESSMENT.lower() or \
+                current_operation == Constants.ASSESSMENT.lower() or \
+                current_operation == Constants.INSTALLATION.lower():
```

Comment:
Assessment and installation checks are handled in later if statements, we don't need those checks in this if

Reply:
Isn't the purpose of this to ensure that there is a ConfigurePatchingResult error added no matter the operation? If so, it seems correct to me.

Reply:
There is code in CoreMain.py to handle ConfigurePatchingResult substatus for failed Assessment and Installation operations.

Here, we only need to add error message to the specific operation where an error is discovered and update it's status to Error. Handling uber level operation or any other operation related to the current one, should already be done by existing code